docs/sdk/core/app1.md

```diff
@@ -15,7 +15,7 @@ Developers can create messages by implementing the `Msg` interface:
 ```go
 type Msg interface {
     // Return the message type.
-    // Must be alphanumeric or empty.
+    // Must be alphabetic or empty.
```

Comment:
I think we're changing this back to be alphanumeric.

Reply:
Should probably change this back when that happens then. Who knows when that will be implemented and merged. I think the docs should be correct in the meantime

Reply:
Just made a PR for this, since it was a pretty easy change. https://github.com/cosmos/cosmos-sdk/pull/1705